include/openssl/core_names.h

```diff
@@ -67,6 +67,8 @@ extern "C" {
 #define OSSL_CIPHER_PARAM_AEAD_IVLEN         OSSL_CIPHER_PARAM_IVLEN
 #define OSSL_CIPHER_PARAM_AEAD_TAGLEN        "taglen"     /* size_t */
 #define OSSL_CIPHER_PARAM_RANDOM_KEY         "randkey"    /* octet_string */
+/* For passing the AlgorithmIdentifier parameter in DER form */
+#define OSSL_CIPHER_PARAM_ALG_ID             "alg_id_param" /* octet_string */
```

Comment:
Add documentation for this..
It should probably mention that it needs to be called twice when setting.. (once to determine the size)

Reply:
That mechanism should be better documented.